### PR TITLE
release v3.0.0

### PR DESCRIPTION
**Pull request by adonesky1**
### Changed
- **BREAKING:**  Bump required node version from v12 to v16 ([#85](https://github.com/MetaMask/mobile-provider/pull/85))
- Bump version of `@metamask/inpage-provider` to `v13.0.0` (package renamed to `@metamask/providers`) ([#85](https://github.com/MetaMask/mobile-provider/pull/85))
- Bump version of `stream-browserify` to `v3.0.0` ([#85](https://github.com/MetaMask/mobile-provider/pull/85))

### CODE REVIEW

File: CHANGELOG.md
Comment (jiexi) [inline CHANGELOG.md:1]: think we need to update the diff links at the very bottom of this file

Comment (adonesky1) [reply CHANGELOG.md:1]: fixed

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +44,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `@metamask/inpage-provider@6.1.0` ([#34](https://github.com/MetaMask/mobile-provider/pull/34))
 
-[unreleased]: https://github.com/MetaMask/mobile-provider/compare/v2.1.0...HEAD
+[unreleased]: https://github.com/MetaMask/mobile-provider/compare/v3.0.0...HEAD
+[2.0.1]: https://github.com/MetaMask/mobile-provider/compare/v2.1.0...v3.0.0
 [2.0.1]: https://github.com/MetaMask/mobile-provider/compare/v2.0.1...v2.1.0
```
Comment (jiexi) [inline CHANGELOG.md]:
```suggestion
[3.0.0]: https://github.com/MetaMask/mobile-provider/compare/v2.1.0...v3.0.0
[2.1.0]: https://github.com/MetaMask/mobile-provider/compare/v2.0.1...v2.1.0
```

**Review by jiexi — APPROVED** [review]

**Review by Cal-L — APPROVED** [review]
Lgtm

**Pull request merged**